docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -29,51 +29,57 @@ loki.source.file "LABEL" {
 ```
 
 ## Arguments
+
 The component starts a new reader for each of the given `targets` and fans out
 log entries to the list of receivers passed in `forward_to`.
 
 `loki.source.file` supports the following arguments:
 
- Name         | Type                 | Description                                                | Default | Required 
---------------|----------------------|------------------------------------------------------------|---------|----------
- `targets`    | `list(map(string))`  | List of files to read from.                                |         | yes      
- `forward_to` | `list(LogsReceiver)` | List of receivers to send log entries to.                  |         | yes      
- `encoding`   | `string`             | The encoding to convert from when reading files.           | `""`    | no       
+| Name            | Type                 | Description                                                                         | Default | Required |
+| --------------- | -------------------- | ----------------------------------------------------------------------------------- | ------- | -------- |
+| `targets`       | `list(map(string))`  | List of files to read from.                                                         |         | yes      |
+| `forward_to`    | `list(LogsReceiver)` | List of receivers to send log entries to.                                           |         | yes      |
+| `encoding`      | `string`             | The encoding to convert from when reading files.                                    | `""`    | no       |
+| `tail_from_end` | `bool`               | Whether a log file should be tailed from the end if a cached position is not found. | `false` | no       |
```

Comment:
Will users know what 'cached position' means? This is the first mention of it in this page. We should probably use similar vocabulary like the one in 'component behaviour' section.

Reply:
changed to 'stored' to match the vocabulary of the section in the component behavior